src/Raven.Server/Documents/PeriodicBackup/BackupTask.cs

```diff
@@ -488,11 +487,21 @@ private void AddInfo(string message, Action<IOperationProgress> onProgress)
             // the last etag is already included in the last backup
             startDocumentEtag = startDocumentEtag == null ? 0 : ++startDocumentEtag;
 
-            using (var file = File.Open(backupFilePath, FileMode.CreateNew))
+            Stream stream;
+            if (_configuration.EncryptionSettings != null)
+            {
+                var key = _configuration.EncryptionSettings.Key;
+                stream = new EncryptingXChaCha20Poly1305Stream(File.Open(backupFilePath, FileMode.CreateNew), 
+                    Convert.FromBase64String(key), _configuration);
+            }
+            else
+                stream = File.Open(backupFilePath, FileMode.CreateNew);
```

Comment:
Let's wrap this in a using, as it was.
We first create the file stream, then create the encrypting stream
Both of which are in a using statement.